doc/api_entity.md

```diff
@@ -216,17 +216,17 @@ A boolean indicating if validation is enabled or not.
 
 ---------
 
-## .getValidation()
+## .getController() (useless)
```

Comment:
there's currently no use for it. all validation is on the form. adding the controller has no added benefit i could find